library/src/fy_library/constants.py

```diff
@@ -23,3 +23,5 @@
 FY_END_MARKER: Final = f"{' '*8}# fy:end <<<==="
 
 FY_PY_FILE_EXTENSION: Final = "_fy.py"
+
+_SETTER: Final = "setter"
```

Comment:
Make it public variable. Rename it:
```suggestion
PROPERTY_SETTER_IMPLEMENTATION_NAME: Final = "setter"
```

Reply:
Renamed